certified-connectors/SigningHub/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1936 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "SigningHub is a complete solution for document approval workflows, advanced digital signatures and document status tracking. It is designed to quickly optimize the way businesses deliver, review, approve and sign their business documents.",
+    "title": "SigningHub",
+    "version": "1.2.0",
+		"contact":{
```

Comment:
@jhan-zaib Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
@jhan-zaib Please fix the alignment of the properties in the swagger.

Reply:
Strange thing, this indentation and alignment changes are made various times but after pushing to GIT, no idea how indentation and alignment disturbed?